client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -3,23 +3,79 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Fragment } from 'react';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
+import Card from 'components/card';
+import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
 
 class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
+	state = {
+		city: '',
+		name: '',
+		stateLoc: '',
```

Comment:
Not sure if we're currently trying to prevent ambiguity by using a key, different than `state`? Wondering if we can pick a better name for `stateLoc`. How about `stateName` or just `state`?

Reply:
Yes, I'd prefer to avoid it. `stateName` it is! 👍